elodie/tests/plugins/googlephotos/googlephotos_test.py

```diff
@@ -34,6 +37,16 @@
 sample_metadata = sample_photo.get_metadata()
```

Comment:
Looks like [Exiftool isn't running](https://travis-ci.org/jmathai/elodie/jobs/636196856#L1850) when getting the metadata. This error is via python3.

```
  File "/home/travis/build/jmathai/elodie/elodie/tests/plugins/googlephotos/googlephotos_test.py", line 37, in <module>
    sample_metadata = sample_photo.get_metadata()
  File "/home/travis/build/jmathai/elodie/elodie/media/base.py", line 93, in get_metadata
    'date_taken': self.get_date_taken(),
  File "/home/travis/build/jmathai/elodie/elodie/media/photo.py", line 61, in get_date_taken
    exif = self.get_exiftool_attributes()
  File "/home/travis/build/jmathai/elodie/elodie/media/media.py", line 122, in get_exiftool_attributes
    exif_metadata = ExifTool().get_metadata(source)
  File "/home/travis/build/jmathai/elodie/elodie/external/pyexiftool.py", line 338, in get_metadata
    return self.execute_json(filename)[0]
  File "/home/travis/build/jmathai/elodie/elodie/external/pyexiftool.py", line 320, in execute_json
    return json.loads(self.execute(b"-j", *params).decode("utf-8"))
  File "/home/travis/build/jmathai/elodie/elodie/external/pyexiftool.py", line 283, in execute
    raise ValueError("ExifTool instance not running.")
ValueError: ExifTool instance not running.
```

Reply:
Moved some of the global variables back into the test functions so ExifTool could be initialized by setup_module before tests run.